jsk_perception/sample/sample_darknet_yolo_object_detection.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
```

Comment:
What about naming `sample_darknet_object_detection.launch`, and adding test named `darknet_object_detection.test` in tests directory as for other nodes.